modules/core/src/test/java/org/apache/ignite/util/GridCommandHandlerAbstractTest.java

```diff
@@ -200,4 +209,48 @@ protected void addExtraArguments(List<String> args) {
     protected void injectTestSystemOut() {
         System.setOut(new PrintStream(testOut));
     }
+
+    /**
+     * Checks if all non-system txs and non-system mvcc futures are finished.
+     */
+    protected void checkUserFutures() {
+        for (Ignite ignite : G.allGrids()) {
+            IgniteEx ig = (IgniteEx)ignite;
+
+            final Collection<GridCacheFuture<?>> futs = ig.context().cache().context().mvcc().activeFutures();
```

Comment:
Please add more descriptive variable name. mvccFuts of something like this

Reply:
fixed